src/models/Factor.js

```diff
@@ -161,10 +171,10 @@ function (Okta, Q, factorUtil, BaseLoginModel) {
           return (provider === 'OKTA' && _.contains(['push', 'call'], factorType));
```

Comment:
Talked to @nandagopalseshagiri-okta and @mauriciocastillosilva-okta  offline. SMS will also have transaction.resend soon. We are adding it to email to keep it consistent.